includes/manifest.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * HM Platform Plugin Manifest.
+ */
+
+namespace HM\Platform;
+
+// Cavalcade.
+Plugin::register( 'cavalcade', 'plugins/cavalcade/plugin.php' )
+      ->load_on( true )
+      ->load_with( function ( $plugin ) {
+	      // Load the Cavalcade Runner CloudWatch extension.
+	      // This is loaded on the Cavalcade-Runner, not WordPress, crazy I know.
+	      if ( class_exists( 'HM\\Cavalcade\\Runner\\Runner' ) && HM_ENV_TYPE !== 'local' ) {
+		      require_once ROOT_DIR . '/lib/aws-sdk/aws-autoloader.php';
+		      require_once ROOT_DIR . '/lib/cavalcade-runner-to-cloudwatch/plugin.php';
+	      }
+
+	      // Load plugin on normal hook.
+	      add_action( 'muplugins_loaded', function () use ( $plugin ) {
+
+		      // Force DISABLE_WP_CRON for Cavalcade.
+		      if ( ! defined( 'DISABLE_WP_CRON' ) ) {
+			      define( 'DISABLE_WP_CRON', true );
+		      }
+
+		      require $plugin->get_file();
+	      } );
+      } )
+      ->enabled( true )
+      ->set_data( function () {
```

Comment:
Why does `set_data` take a function? Do we actually _need_ to do that?

Reply:
Because it runs so early and I wanted it to be translatable, however I think I can get this from the docs API now instead